frontends/web/src/routes/account/send/send.tsx

```diff
@@ -767,6 +767,7 @@ class Send extends Component<Props, State> {
                                     <p>
                                         <span key="proposedAmount">
                                             {(proposedAmount && proposedAmount.amount) || 'N/A'}
+                                            {' '}
```

Comment:
i assume this regression happened in https://github.com/digitalbitbox/bitbox-wallet-app/commit/d1bbaa2edbd62f8378046ea1024e16ee3b4b172e, when this was split into multiple lines.

do you remember splitting a single line on multiple lines in other places? maybe the same kind of bug also appears elsewhere.